rust/src/http2/range.rs

```diff
@@ -131,11 +131,11 @@ pub fn http2_range_open(
     tx: &mut HTTP2Transaction, v: &HTTPContentRange, flow: *const Flow,
     cfg: &'static SuricataFileContext, dir: Direction, data: &[u8],
 ) {
-    if v.end <= 0 || v.size <= 0 {
-        // skipped for incomplete range information
-        return;
-    } else if v.end == v.size - 1 && v.start == 0 {
+    if (v.end <= 0 || v.size <= 0)
+        // skipped for incomplete range information 
+    || (v.end == v.size - 1 && v.start == 0)
         // whole file in one range
+    {
```

Comment:
There is nothing technically wrong with this, but after looking a little closer, I think there is a better way that makes this more redable.

Instead of a combined if/else statement, just conver to 2 if statements on their own.  It'll pass the lint and be more readable.

Reply:
Do you mean like this: 
``` rust
if v.end <= 0 || v.size <= 0 {
       // skipped for incomplete range information 
       return;
 } if v.end == v.size - 1 && v.start == 0{
       // whole file in one range
       return:
}
```

Reply:
Yes, but with a line break between each if :P

Reply:
ok, Thanks :)